utils/_utils/geocodeLocation.js

```diff
@@ -0,0 +1,39 @@
+import * as Location from "expo-location";
+
+export const convertToCoordinates = (address, success) => {
+
+    Location.geocodeAsync(address)
+    .then((results) => {
+        const coordinates = { lat: results[0].latitude, lng: results[0].longitude }
+
+        // accepts a success callback, otherwise just returns the value
+        if(success) success(coordinates)
+        return coordinates
+    })
+    .catch((error) => {
+        console.error('cannot geocode address!', {error})
+    })
+}
+
+export const convertToAddress = (coordinates, success) => {
+
+    let coords = {coordinates}
+    if (coordinates && coordinates.lat || coordinates.lng) {
+        coords = { latitude: coordinates.lat, longitude: coordinates.lng }
+    }
+    else coords = { latitude: coordinates.latitude, longitude: coordinates.longitude }
+
+    Location.reverseGeocodeAsync(coords)
+    .then((results) => {
+        // street, name, city, region, country, postalCode, name
+        const { name, city, region, country } = results[0]
+        const address = `${name}, ${city} ${region}, ${country}`
```

Comment:
name seems to be the house number as far as i know. We may discuss the actual format we want to display to the user later. I believe name street city would work for now

Reply:
Sure!

Reply:
Yea it was showing 5755 Dalhousie Road as 5755, Vancouver, B.C., Canada 😂